File: Exam2/C++/square.cpp
#include "square.h"
#include <iostream>
#include <string>

using namespace std;

Square::Square(string aColor, float aBorder, float aSide)
{
	color = aColor;
	border = aBorder;
	side = aSide;
}

string Square::getColor()
{
		return color;
}

float Square::getBorder()
{
	return border;
}

float Square::getSide()
{
	return side;
}

void Square::draw()
{
	cout << getColor() << " square of side "<< getSide() << " and border " << getBorder() << endl;

}

File: Exam2/Ruby/square.rb
# square

require "shape"

class Square < Shape

	def initialize(aColor, aBorder, aSide)
	super(aColor, aBorder)
	@side = aSide
	
	end

	def draw
		puts "#{getColor()} square of side #{getSide()} and border #{getBorder()}."
	end

	def getSide
		@side
	end

end
